servers/quarkus-server/pom.xml

```diff
@@ -279,6 +279,8 @@
               <filesets>
                 <fileset>
                   <directory>${project.build.directory}/rocksdb-test</directory>
```

Comment:
Is the `rocksdb-test` directory still used now?

Reply:
Not really, but I kept it just in case.